### PR TITLE
Added Home Remedies cheat sheet

### PR DESCRIPTION
**Pull request by jcode82**
homeRemedies is a useful list of links to help users in the search for everyday home remedies. This IA is divided up into several categories to give the user options on the way they want to go about their search, for example: Websites, Videos, Books, Blogs and Podcasts.
unspecified

---

IA Page: https://duck.co/ia/view/home_remedies_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@jcode82 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@jcode82 thanks for this! 

Don't forget to link the IA Page in your PR description. That's required for use to deploy this to beta, test it, and eventually merge it :+1:

**Comment by jcode82** [comment]
Oh yes, just filled out and submitted the IA page. I look forward to hear from you soon. 

Juan F

**Comment by jcode82** [comment]
IA: https://duck.co/ia/view/home_remedies

**Comment by alohaas** [comment]
@jcode82, thanks again for creating the Home Remedies Cheat Sheet! You’ve been doing a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in home remedies and wellness. 

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to home remedies and wellness.

OR

- Test live wellness-related Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

File: share/goodie/cheat_sheets/json/home-remedies.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  "id": "home_remedies_cheat_sheet",
+  "name": "homeRemedies",
+  "description": "At home treatments that may contain spices, vegetables, or other common items",
+  "metadata": {
+    "sourceName": "Home Remedies",
+    "sourceUrl": "https://en.wikipedia.org/wiki/Traditional_medicine"
+  },
+   "aliases": [
+        "remedies",
+        "first aid",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/home-remedies.json]: Lets remove this one :wink:

**Comment by moollaza** [comment]
@jcode82 Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

It looks like all that's needed for merge is to remove that "first aid" alias

File: share/goodie/cheat_sheets/json/home-remedies.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{
+  "id": "home_remedies_cheat_sheet",
+  "name": "homeRemedies",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/home-remedies.json]: Lets change this to `Home Remedies`

File: share/goodie/cheat_sheets/json/home-remedies.json [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  "description": "At home treatments that may contain spices, vegetables, or other common items",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/home-remedies.json]: Use `Home treatments...`

File: share/goodie/cheat_sheets/json/home-remedies.json [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+  "metadata": {
+    "sourceName": "Home Remedies",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/home-remedies.json]: `Wikipedia`

File: share/goodie/cheat_sheets/json/home-remedies.json [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    "sourceUrl": "https://en.wikipedia.org/wiki/Traditional_medicine"
+  },
+   "aliases": [
+        "remedies",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/home-remedies.json]: I think this one is too broad (law, standard medicines etc.)

File: share/goodie/cheat_sheets/json/home-remedies.json [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+    ],
+    "sections": {
+        "Websites": [
+            {
+                "link": "http://www.webmd.com/cold-and-flu/cold-guide/cold-guide-support-resources",
+                "key": "Cold, Flu, & Cough Health Center"
+            },
+            {
+                "link": "https://en.wikipedia.org/wiki/Traditional_medicine",
+                "key": "Traditional medicine"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/home-remedies.json]: Perhaps `Traditional Medicine (Wikipedia)`

**Comment by pjhampton** [comment]
This PR looks like it has went stale over time. Closing [for now]

**Pull request closed without merging**